### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Scribe/Scribe-i18n](https://hosted.weblate.org/projects/scribe/scribe-i18n/).

Current translation status:

<a href="https://hosted.weblate.org/engage/scribe/">
Translation status
</a>

### CODE REVIEW

File: Scribe-i18n/sv.json [hunk 1]
```diff
@@ -1,57 +1,60 @@
...
+    "app.settings.layout.periodAndComma": "Punk och komma på ABC",
+    "app.settings.layout.periodAndComma.description": "Inkludera komma- och punkttangenter på huvudtangentbordet för att underlätta skrivandet.",
+    "app.settings.title": "Inställningar",
+    "app.wikidataExplanation1": "Wikidata är en gemensamt redigerad kunskapsgraf som underhålls av Wikimedia Foundation. Det fungerar som en källa till öppen data för projekt som Wikipedia och flera andra.",
+    "app.wikidataExplanation2": "Scribe använder Wikidatas språkdata för många av sina kärnfunktioner. Vi får information som substantiv, genus, verbböjningar och mycket mer!",
+    "app.wikidataExplanation3": "Du kan skapa ett konto på wikidata.org för att gå med i communityn som stöder Scribe och så många andra projekt. Hjälp oss att ge gratis information till världen!",
+    "app.about.appHint": "Här kan du lära dig mer om Scribe och dess community.",
+    "app.installation.appHint": "Följ instruktionerna nedan för att installera Scribe-tangentbord på din enhet.",
+    "app.settings.appHint": "Inställningar för appen och installerade tangentbord finns här."
```
Comment (wkyoshida) [inline Scribe-i18n/sv.json:59]: My guess is these keys were added after the `sv.json` file was created? Which might be why Weblate simply appended them to the end of the json body

Made me wonder if there's a way for us to configure Weblate to sort the keys for us automatically. Looks like there might be!? :eyes::thinking: There's a [`Customize JSON output`](https://docs.weblate.org/en/latest/admin/addons.html#customize-json-output) addon that seems like we could activate? Might need to be an admin on the Weblate project to add it though

**Review by wkyoshida — APPROVED** [review]
I added a quick note on something we could try on Weblate, but we should just be good with this Weblate PR?

**Comment by andrewtavis** [comment]
Hey @wkyoshida! 👋 Thanks for looking into the sorting! Makes total sense, and I'll look into that. As far as this is concerned, let's wait on @henrikth93 to send along the last two strings and then we'll be good to merge 😊

**Review by andrewtavis — APPROVED** [review]
Thanks so much for this, @henrikth93! Stoked to have two languages ready for v3.1! 🚀

**Pull request merged**